pentestportal_sg_rules.tf

```
@@ -57,8 +65,8 @@ resource "aws_security_group_rule" "pentestportal_ingress_from_kali_via_8080" {
 # Allow egress to anywhere via HTTP and HTTPS
 # For: Pentest portal installation dependencies
 resource "aws_security_group_rule" "pentestportal_egress_to_anywhere_via_allowed_ports" {
+  for_each = toset([for port in ["80", "443"] : port if lookup(var.operations_instance_counts, "pentestportal", 0) > 0])
```

More efficient would be
```suggestion
  for_each = lookup(var.operations_instance_counts, "pentestportal", 0) > 0 ? toset(["80", "443"]) : toset([])
```